tools/Unix.mk

```diff
@@ -328,13 +328,20 @@ arch/dummy/Kconfig:
 	@echo "CP: $@ to $(CHIP_KCONFIG)"
 	$(Q) cp -f $(CHIP_KCONFIG) $@
 
+# Copy $(BOARD_KCONFIG) to boards/dummy/Kconfig
+
+boards/dummy/Kconfig:
+	@echo "CP: $@ to $(BOARD_KCONFIG)"
+	$(Q) cp -f $(BOARD_KCONFIG) $@
```

Comment:
@AlanRosenthal, please don't forget to mirror these changes to the Win.mk file.

Reply:
will do, looks like there's some issues with CI, once I have CI passing I'll mirror the changes to Win.mk